build/configs/integreat/index.js

```diff
@@ -1,14 +1,16 @@
 // @flow
 
-import type { AppConfigType } from '../../src/AppConfigType'
-
-import brightTheme from '../../src/modules/theme/constants/theme'
+import integreatTheme, { darkTheme as darkIntegreatTheme } from '../../themes/integreat'
+import type { AppConfigType } from '../AppConfigType'
+import featureFlags from '../featureFlags'
 
 const IntegreatAppConfig: AppConfigType = {
```

Comment:
The term `AppConfig` should not be used anymore imo. As stated in the issue the term is `BuildConfig`. Each build config contains Feature Flags.